### PR TITLE
update package

### PR DESCRIPTION
**Pull request by JudahSchwartz**
demisto-sdk nightly https://code.pan.run/xsoar/content/-/pipelines/4599774
Content nightly https://code.pan.run/xsoar/content/-/pipelines/4599773
Private https://github.com/demisto/content-private/pull/652

gold branch https://code.pan.run/xsoar/content-internal-dist/-/merge_requests/219
run: https://code.pan.run/xsoar/content-internal-dist/-/pipelines/4600958

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/CrowdStrikeFalconSandbox/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "CrowdStrike Falcon Sandbox",
     "description": "Fully automated malware analysis (formerly Payload Security VxStream).",
     "support": "xsoar",
-    "currentVersion": "2.0.7",
+    "currentVersion": "2.0.6",
```
Comment (dorschw) [inline Packs/CrowdStrikeFalconSandbox/pack_metadata.json:5]: ?

Comment (JudahSchwartz) [reply Packs/CrowdStrikeFalconSandbox/pack_metadata.json]: accident. removed

**Review by dorschw — CHANGES_REQUESTED** [review]
Please replace the PR body with links to all nightly runs.

**Comment by JudahSchwartz** [comment]
@dorschw done

**Comment by JudahSchwartz** [comment]
Poetry (version 1.2.0)
.circleci/config.yml is up to date!
poetry.lock is part of the branch changes, proceeding
pyproject.toml is part of the branch changes, proceeding
=== sourcing /tmp/.bash_env-63e8d02be0a87026ed3700d9-0-build ===
Checking if pyproject.toml is consistent with poetry.lock
Creating virtualenv content in /home/circleci/project/.venv

The lock file is not compatible with the current version of Poetry.
Upgrade Poetry to be able to read the lock file or, alternatively, regenerate the lock file with the `poetry lock` command

@dorschw looks like i'll need a second to fix this. @ilaner didnt we already fix this?

**Comment by dorschw** [comment]
> @dorschw done

please add for gold & private

File: Tests/scripts/validate.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 set -e
 
 echo "CI_COMMIT_BRANCH: $CI_COMMIT_BRANCH CI: $CI DEMISTO_README_VALIDATION: $DEMISTO_README_VALIDATION"
-
+npm list --json
```
Comment (JudahSchwartz) [inline Tests/scripts/validate.sh:5]:
```suggestion
```

**Comment by JudahSchwartz** [comment]
upgrading mdx causes more packs to fail validation, but looks like rightfully so

image
https://xsoar.pan.dev/docs/reference/integrations/virus-total---premium-api-v3

**Comment by JudahSchwartz** [comment]
im downgrading mdx for now to see if thats the only reason for the failures. If it is i'll open an issue to upgrade it so it doesnt need to be done in this pr

**Pull request merged**